### PR TITLE
fix(Persona): use `<button/>`

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/PersonaWrap/PersonaWrap.scss [hunk 1]
```diff
@@ -110,31 +110,17 @@ $block: '.#{variables.$ns}persona';
...
-
-        transition-property: color;
-        transition-duration: $transitionDuration;
-        transition-timing-function: $transitionTimingFunction;
-
-        &:hover {
-            color: var(--yc-color-text-primary);
+        @at-root #{&}#{&} {
+            --yc-button-height: #{$avatarSize};
+            --yc-button-background-color-hover: transparent;
```
Comment (amje) [inline src/components/PersonaWrap/PersonaWrap.scss:123]: Can we workaround this? These vars are considered as private

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/PersonaWrap/PersonaWrap.scss (hunk 1 above)
Comment (ogonkov) [reply src/components/PersonaWrap/PersonaWrap.scss:123]: Private even for components of library itself?

**Review by DaffPunks — APPROVED** [review]

File: styles/mixins.scss [hunk 2]
```diff
@@ -19,6 +19,28 @@
     cursor: pointer;
 }
 
+@mixin button-outline {
```
[the remark this answers is not in the document]
Comment (ogonkov) [reply styles/mixins.scss:22]: It is kinda Button-specific, in other components there slightly different implementations of focus ring

File: src/components/Button/Button.scss [hunk 3]
```diff
@@ -55,13 +47,7 @@ $block: '.#{variables.$ns}button';
         background-color: var(--yc-button-background-color-hover);
```
[the remark this answers is not in the document]
Comment (ogonkov) [reply src/components/Button/Button.scss:47]: I think this would cause full button styles refactoring, are we want it in this PR?

**Comment by ogonkov** [comment]
Fixed via #1070

**Pull request closed without merging**